src/viam/sdk/module/service.cpp

```diff
@@ -148,7 +148,7 @@ ::grpc::Status ModuleService_::ValidateConfig(
     }
     try {
         const std::vector<std::string> implicit_deps = reg->validate(cfg);
-        for (auto& dep : implicit_deps) {
+        for (const auto& dep : implicit_deps) {
```

Comment:
This probably should have been in the `const` change. I think it just got added to the wrong commit.